go.mod

```diff
@@ -98,34 +98,34 @@ require (
 	github.com/google/btree v1.1.2
 	github.com/google/go-attestation v0.5.1
 	github.com/google/go-cmp v0.6.0
-	github.com/google/go-containerregistry v0.19.0
+	github.com/google/go-containerregistry v0.19.1
 	github.com/google/go-querystring v1.1.0
 	github.com/google/go-tpm-tools v0.4.3
 	github.com/google/renameio/v2 v2.0.0
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/google/uuid v1.6.0
-	github.com/googleapis/gax-go/v2 v2.12.2
+	github.com/googleapis/gax-go/v2 v2.12.3
 	github.com/gorilla/websocket v1.5.1
-	github.com/gravitational/license v0.0.0-20231228155916-928ed9ac0335
-	github.com/gravitational/oxy v0.0.0-20221029012416-9fbf4c444680
+	github.com/gravitational/license v0.0.0-20240313232707-8312e719d624
+	github.com/gravitational/oxy v0.0.0-20231219172753-f855322f2a6c
```

Comment:
Big jump here. I wonder why this was untouched until now.

Reply:
I am not sure, I don't see it ever being ignored in the history

Reply:
CI is green so it's probably OK, but still it's weird. Maybe someone else has an idea.